content/go/concepts/switch/switch.md

```diff
@@ -0,0 +1,126 @@
+---
+Title: 'Switch'
+Description: 'A switch statement allows the selective execution of one of several blocks of code based on the value of an expression.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comparison'
+  - 'Control Flow'
+  - 'Switch'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+A `switch` statement allows the selective execution of one of several blocks of code based on the value of an expression. It can be thought of as an expanded `if ... else` statement, with many potential `if` clauses.
+
+## Syntax 1
+
+There are many syntaxes that can be used in the `switch` statement. The following is probably the most common syntax:
```

Comment:
```suggestion
There are many syntaxes that can be used in the `switch` statement. The following is probably the most common:
```

Reply:
`switch` the last sentence for "The following are probably" and I think we got a deal! 😄